frontend_app_kanban/i18next.scanner/de/translation.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Error while saving the comment": "__NOT_TRANSLATED__",
+  "Delete": "__NOT_TRANSLATED__",
```

Comment:
"Delete": "Löschen",
  "A content with the same name already exists": "Ein Inhalt mit demselben Namen existiert bereits",
  "Error while creating kanban": "Fehler beim Erstellen von Kanban",
  "Validate and create": "Validieren und erstellen",
  "Please enter the new title of this card": "Bitte geben Sie den neuen Titel für diese Karte ein",
  "You must change the status or restore this kanban board before any change": "Sie müssen den Status ändern oder diese Kanbantafel vor jeder Änderung wiederherstellen",
  "Error while saving the new version": "Fehler beim Speichern der neuen Version",
  "Please enter the new name of the column": "Bitte geben Sie den neuen Namen der Spalte ein",
  "column": "Spalte",
  "Fullscreen": "Vollbild",
  "Board: {{label}}": "Tafel: {{label}}",
  "Loading, please wait…": "Laden, bitte warten...",
  "Error while loading the board.": "Fehler beim Laden des Boards.",
  "Change the color of this card": "Ändern Sie die Farbe dieser Karte",
  "Remove this card": "Diese Karte entfernen",
  "Are you sure you want to delete this card?": "Sind Sie sicher, dass Sie diese Karte löschen möchten?",
  "Editing Card": "Karte bearbeiten",
  "New Card": "Neue Karte",
  "Title:": "Titel",
  "Description:": "Beschreibung",
  "Enable HTML": "HTML aktivieren",
  "Color:": "Farbe",
  "Deadline:": "Deadline",
  "Duration:": "Dauer",
  "Cancel": "Abbrechen",
  "Validate": "Bestätigen",
  "Change the color of this column": "Ändern Sie die Farbe dieser Spalte",
  "Add a card": "Eine Karte hinzufügen",
  "Remove this column": "Diese Spalte entfernen",
  "Are you sure you want to delete this column?": "Sind Sie sicher, dass Sie diese Spalte löschen möchten?",
  "Hide": "Ausblenden",
  "Timeline": "Zeitleiste",
  "Tags": "Markierungen",
  "New Kanban board": "Neue Kanban-Tafel",
  "Board's name": "Name der Tafel"